src/ssl.c

```diff
@@ -16570,9 +16574,23 @@ WOLFSSL_BIGNUM *wolfSSL_ASN1_INTEGER_to_BN(const WOLFSSL_ASN1_INTEGER *ai,
         return NULL;
     }
 
-    if (SetIndividualExternal(&bn, &mpi) != SSL_SUCCESS) {
-        return NULL;
+    /* SetIndividualExternal mallocs bn in the case that bn is NULL
+     * mp_clear needs called because mpi is copied and causes memory leak with
+     * --disable-fastmath */
+    if (bn == NULL) {
+        if (SetIndividualExternal(&bn, &mpi) != SSL_SUCCESS) {
+            wolfSSL_BN_free(bn);
```

Comment:
Will move the bn free to SetIndividualExternal. With mp_clear I can do that by making use of the ret variable. Will update it.